### DISCUSSION TITLE
2006/10/18/%E3%80%8A%E7%AE%A1%E7%90%86%E8%87%AA%E5%B7%B1%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#122] [Announcements]
# 2006/10/18/%E3%80%8A%E7%AE%A1%E7%90%86%E8%87%AA%E5%B7%B1%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

这篇《管理自己》是我在《商业评论》期刊上读到的一篇文章，它节选自彼得•德鲁克（Peter F. Drucker）的《21世纪的管理挑战》一书。

https://blog.qizhen.xyz/2006/10/18/%E3%80%8A%E7%AE%A1%E7%90%86%E8%87%AA%E5%B7%B1%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：


竹贤
2006年10月20日 16:36 编辑
立足当前、着眼未来，量力而行、尽力而为。

回复
竹贤
2006年10月20日 10:37 编辑
你文中所说的“技术”和“管理”的范畴都是狭义的词汇。其实，在今天的社会里，已经很少有人能够从事这种单纯的“技术”或“管理”了。
即使你是一个刚出道的纯技术人员，在日常工作中你也得与你的同事、上司、客户等不断打交道，你得安排你的工作，希望提高工作效率，期望得到上司和同事的认可，这里就已经包含了各种各样的“管理”在内。假如你多少有了一点资历和经验，你可能要带领几位同事一起完成某项工作，从此你就已经与“管理”紧密联系在一起了。
假如你是一个“管理”人员，即使你不是技术开发部或营销部的主管，不过是一位人力资源部、或财务部、或后勤等杂务的一般工作人员，你又希望你的工作出色，那你就得知道一点管理方面的专业知识，其实这也是一门技术。更不用说，为了你的升迁，你就得多少懂得一些与你公司业务有关的专业技术了。懂得越多，对你的工作就越有帮助。
“管理”是一门科学，是一种艺术，是一种实践。无论你将来从事什么，无论你对它有无兴趣,都可以也应该学学。更主要要有实践，学了就用。纸上谈兵对你我等人来说大概是毫无用处的。
究竟是偏重“技术”还是偏重“管理”，这要看自己的特长和兴趣，还要看社会的需要，更要看机遇。对自己来说，多学一门知识，增加一些才能，就行。不要把学习仅仅当作一种谋生的手段，它应该是我们人生旅途中必不可少的需求。

回复
奕瑋
2006年10月19日 11:57 编辑
我现在是脚踩两只船，管理和技术都做，但明显的力不从心。
从小的精英教育模式，使我什么都想做好，可我毕竟是普通人。
想过将来读个哲学博士，再后来当个客座教授什么的，
那些大概需要丰富的人生阅历。
我会一直去努力，无论是否来得及。

回复
Qizhen
2006年10月19日 09:48 编辑
收到的一条反馈意见：

“无论是从事专业技术工作，还是从事管理工作，首要一条：先真正“管理自己”。如果连自己都管理不好，如何管理好一个组织、一个部门甚至一个企业呢？
你没有从事过管理，对管理的难度并不清楚。如果单纯为了熬到一个“基层管理者”，还不如踏踏实实地做一个基层的工程师吧——毕竟你还熟悉一点。
管理是和人打交道，也是一门更专业的技术工作。它又是一门艺术，要求从事者有亲和力、有独特的人格魅力、善于与人相处、善闻善问，它要求你博闻强记、要求你懂得人文社会心理等多方面的知识。
人员的结构都是金字塔型的——极大多数都是在它的底层，经过自己的努力奋斗，又有难得的机遇，最终达到顶点的只是少数。
目标是要有的，努力更是不可缺少的，至于将来具体的每一步怎么走，有的时候是无法预料的。你现在还需要要有的是“只管耕耘，不问收获”的一种平和心态——这与远大的理想目标并不矛盾。”

回复
永青
2006年10月18日 21:36 编辑
这篇文章选自《21世纪的管理挑战》的第六章
这本书我有，如果需要，随时说一声

回复
